### ISSUE TITLE
[Feature Request]  Auto-translate but only for TTS input.

### ISSUE THREAD

**Issue by user177013** [SillyTavern/SillyTavern#1119]
**Have you searched for similar [requests](https://github.com/SillyTavern/SillyTavern/issues?q=)?**
Yes

**Is your feature request related to a problem? Please describe.**
When you want to use TTS in different language than messages, you need to use auto-translate or manually hitting the translate button manually, then it print the message in translated language and pass it to TTS. This results to user having to hit the button back and forth.
Basically an option to bypass printing the translated message and instead just pass it to TTS in background.

**Describe the solution you'd like**
Usually it goes: `message>translate>print the translation>TTS`
What should be an option: `message>translate>TTS`

**Comment by xNissX233** [comment]
This seems similar to what happens in CharacterAI. Your character says something in English, but if the voice is in Japanese, he well speak the English message in Japanese. It's a genius feature and I was surprised when I saw it in CharacterAI, I wish this could be implemented here too.
